src/utils/internal/getCallFrame.test.ts

```diff
@@ -0,0 +1,163 @@
+import { getCallFrame } from './getCallFrame'
```

Comment:
This test looks great! 
We should also have the same test suite for NodeJS. Stack traces there may differ, it would be nice to test them.

```js
// src/utils/internal/getStackFrame.node.test.ts
/**
 * @jest-environment node
 */
```

Would you be interested in adding the NodeJS tests? If you don't have time for this I can give you a hand.

Reply:
Hm, this is just a unit test with a mock for `new Error` ... using fake stack traces to simulate NodeJS from both macOS and Windows.

And a real native NodeJS `new Error().stack` is used in the integration test `setup-server/printHandlers.test.ts` already. IMHO no need to test mock errors in 2 different jest environments..

Reply:
Oh, I see. Since we are using an exact system-specific stack trace there's no need to re-create this test suite in a different environment. Nice catch!